src/cript/nodes/core.py

```diff
@@ -82,12 +82,12 @@ def __init__(self, **kwargs):
                         possible_array = getattr(self._json_attrs.kwarg[: -len("_count")])
                         if not isinstance(possible_array, list):
                             raise CRIPTExtraJsonAttributes(self.node_type, kwarg)
-                    except KeyError:
+                    except AttributeError:
```

Comment:
can this error only be raised for too many or too few arguments or do you think it can be raised other places as well?

because if it is only for too few or too many arguments then we could give an error that looks like
```
ImproperNodeCreation: 'Data' node 'my data node name' was created with too few or too many arguments
```

however, if this error can be raised for a lot of reasons and we don't know, then we can give a generic error and that should help them with debugging

Reply:
okie dokes, I can do that

Reply:
actually looking at the error message I think it is okay for now, but I wrote a small trello ticket with ideas for future upgrades

https://trello.com/c/M1JuhAUv